### DISCUSSION TITLE
Jalaali and Gregorian functions

### DISCUSSION THREAD

**Discussion by ghonchesefidi** [persian-tools/persian-tools#101] [General]
I can work on Write Jalaali and Gregorian functions to convert the Date together task that you had mentioned in the readme file.
what methods should I start first?

**Comment by ghonchesefidi** [comment]
Are we going to implement jalali-moment methods or do we need to implement them natively?

**Reply by ali-master** [reply]
Hi there,

I did it before for another OOS project. If you want, you can migrate codes to this project.

**Reply by ghonchesefidi** [reply]
That sounds good! you did most of the work! How can we start then?
